### PR TITLE
Change lsblk usage in linux filesystem plugin

### PR DESCRIPTION
**Pull request by josqu4red**
Fixes issue #469

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @josqu4red

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

File: spec/unit/plugins/linux/filesystem_spec.rb [hunk 1]
```diff
@@ -189,6 +190,7 @@
 /dev/mapper/sys.vg-usr.lv: TYPE=\"ext4\" 
 /dev/mapper/sys.vg-var.lv: TYPE=\"ext4\" 
 /dev/mapper/sys.vg-home.lv: TYPE=\"xfs\" 
+/dev/mapper/debian--7-root: TYPE="ext4"
```
Comment (jaymzh) [inline spec/unit/plugins/linux/filesystem_spec.rb]: you're missing the backslashes.

**Comment by jaymzh** [comment]
seems reasonable.

**Comment by thommay** [comment]
seems legit :+1:

**Comment by josqu4red** [comment]
Anything blocking merge ?

**Pull request merged**